Kickstarter-iOS/AppDelegate.swift

```diff
@@ -97,9 +97,17 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self?.goToLiveStream(project: $0, liveStreamEvent: $1, refTag: $2)
     }
 
+    self.viewModel.outputs.goToCreatorMessageThread
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.goToCreatorMessageThread($0.0, $0.1)
```

Comment:
Thank you Stephen. I will try. I think I actually tried but because of the nullable XCode was complaining. I will double-check.